docs/plugins.rst

```diff
@@ -1,260 +1,133 @@
 Plugins
 =======
 
+A chat bot is meaningless unless you can extend/customize it to fit your own use cases, which can be achieved through custom plugins.
 
-Plugins
--------
+Writing your first plugin
+-------------------------
 
-A chat bot is meaningless unless you can extend/customize it to fit your own use cases.
+#. First you need to create a Python file in the `./plugins` directory. For this example we will create the file ./plugins/my_plugin.py and
```

Comment:
Since a custom bot does not require checking out the source code (I'd even recommend against it), there won't be a plugins directory. Users can just subclass `Plugin` wherever they want and pass it to their `Bot` constructor (as with the settings)